### ISSUE TITLE
Automatic rect/box not drawing manual drawing works [bug]

### ISSUE THREAD

**Issue by armeya25** [louisnw01/lightweight-charts-python#344]
### Question

i am trying to draw box in chart
but it shows nothing
tried changing color, width but no effect,

if anyone have proper working example plz share

thanks in advance

### Code example

```python
from lightweight_charts import Chart
import pandas as pd

chart = Chart()

df = pd.read_csv('symbols/BNF/historical-data/df1m_1.csv')

chart.set(df)
chart.box(df.iloc[-10]['date'], df.iloc[-10]['open'], df.iloc[-3]['date'], df.iloc[-3]['close'], color='white', width=4)

chart.show(block=True)
```

**Comment by cilindro5432** [comment]
Box or rectangle is in toolbox just add

chart = Chart(toolbox=True)
https://lightweight-charts-python.readthedocs.io/en/latest/index.html
read carefully the documentation

**Comment by armeya25** [comment]
Thank you very much

**Comment by armeya25** [comment]
Didn't use toolbox anywhere so I overlooked it's docks

**Comment by armeya25** [comment]
from lightweight_charts import Chart
import pandas as pd

chart = Chart(toolbox=True)

df = pd.read_csv('symbols/BNF/historical-data/df1m_1.csv')

chart.set(df)
chart.box(df.iloc[-10]['date'], 180, df.iloc[-3]['date'], 190, color='#C70039', width=1)

chart.show(block=True)

**Comment by armeya25** [comment]
i am using 2.0 version

**Comment by cilindro5432** [comment]
You can see rectangle symbol on tool box? If you see the rectangle on tool box is version 2.0 if not is the original branch. But if you see the rectangle and not draw may be is a bug.

**Comment by cilindro5432** [comment]
you dont need this
chart.box(df.iloc[-10]['date'], 180, df.iloc[-3]['date'], 190, color='#C70039', width=1)
 in your code to get the rectangle

**Comment by armeya25** [comment]
> You can see rectangle symbol on tool box? If you see the rectangle on tool box is version 2.0 if not is the original branch. But if you see the rectangle and not draw may be is a bug.

I see rectangle in toolbox but it didn't draw rectangle on chart automatically only manual drawing works.

I also changed values so that they should be on visible range but no rectangle apeared

**Comment by louisnw01** [comment]
Fixed in latest 2.0 version.

Louis
